internal/pkg/cli/run_local.go

```diff
@@ -262,14 +286,28 @@ func (o *runLocalOpts) Execute() error {
 		return fmt.Errorf("get task: %w", err)
 	}
 
+	if o.proxy {
+		if err := validateMinEnvVersion(o.ws, o.envChecker, o.appName, o.envName, "v1.32.0", "run local --proxy"); err != nil {
```

Comment:
I feel like we should put all of these min versions - in this case, v1.32.0 - in, maybe, `template/env.go`. Same for [the](https://github.com/aws/copilot-cli/blob/ebe9289dd20ed5f6605d0678337331b7549f76ee/internal/pkg/cli/secret_init.go#L262)